packages/app/src/components/pages/Home/index.js

```diff
@@ -53,4 +53,4 @@ const Home = () => {
   );
 };
 
-export default Home;
+export default Home;
```

Comment:
Nitpick: Please add this newline back.

Reply:
whoops on it

Reply:
Did you add the newline back? I'm still seeing that ⛔️ in GitHub.